tests/Client/UpdaterTest.php

```diff
@@ -1138,6 +1141,10 @@ public function providerFileNotFoundExceptions(): array
             ],
             [
                 'TUFTestFixtureSimple',
+                // Deleting timestamp.json and 1.snapshot.json from the server will cause Updater::updateTimestamp()
+                // and Updater::refresh() to error out. That's fine in these cases, because we're not trying to finish
+                // the refresh. This will implicitly check that Updater::updateRoot() doesn't erroneously think that
+                // keys have been rotated, and therefore delete the local timestamp.json and snapshot.json.
```

Comment:
Since we have separate test method for this now maybe we could have very short with a @see to the new test method?

Reply:
I kept it as-is, but added the `@see`. I figure this is something that is complex enough to justify explaining it verbosely.